slash/react/src/Form/Select/SelectBase.tsx

```diff
@@ -7,7 +7,7 @@ import {
 import { getComponentClassName } from "../../utilities";
 
 type Props = ComponentPropsWithoutRef<"select"> & {
-  options: OptionHTMLAttributes<HTMLOptionElement>[];
+  options?: OptionHTMLAttributes<HTMLOptionElement>[];
```

Comment:
Marquer `options` comme déprécié au profit de children ?

```suggestion
  /**
   * @deprecated Use `children` instead
   * Instead of
   * ```jsx
   * <SelectBase options={[{ value: "1", label: "Option 1" }]} />
   * ```
   * you can now do something like :
   * ```jsx
   * <SelectBase>
   *   <option value="1">Option 1</option>
   * </SelectBase>
   * ```
   * It allows you to use the `optgroup` tag for example.
   */
  options?: OptionHTMLAttributes<HTMLOptionElement>[];
```

Reply:
il faudrait aussi mettre dans le Select et SelectInput du coup.

Reply:
Pour le SelectInput ce sera fait dans une autre PR car les children du select input sont utilisés et affiché après le select

Reply:
options passé en deprecated